contents/data-display/charts/temperature-chart/custom-radio.tsx

```diff
@@ -0,0 +1,67 @@
+import type { UseRadioGroupProps, UseRadioGroupReturn } from "@yamada-ui/react"
+import {
+  Box,
+  HStack,
+  Input,
+  useRadio,
+  useRadioGroup,
+  Text,
+} from "@yamada-ui/react"
+import type { FC } from "react"
+
+type CustomRadioProps = ReturnType<UseRadioGroupReturn["getRadioProps"]> & {
+  currentTemperature: number
+}
+
+const CustomRadio: FC<CustomRadioProps> = ({
+  currentTemperature,
+  ...props
+}) => {
+  const { getInputProps, getIconProps } = useRadio(props)
+
+  return (
+    <Box as="label">
+      <Input {...getInputProps()} />
+
+      <Box
+        {...getIconProps()}
+        cursor="pointer"
+        borderLeftWidth="1px"
+        p="lg"
+        _checked={{
+          bg: ["blackAlpha.50", "whiteAlpha.100"],
+        }}
```

Comment:
I think you should add a hover style to the radio trigger button.
```suggestion
        }}
        _hover={{
          bg: ["blackAlpha.50", "whiteAlpha.100"],
        }}
```

Reply:
I don't think it is necessary because it will be difficult to see.
I refer to `Bar Chart - Interactive`.
https://ui.shadcn.com/charts